frontend/app/src/shared/components/form/utils/getParentRelationship.ts

```diff
@@ -0,0 +1,11 @@
+import { useSchema } from "@/entities/schema/ui/hooks/useSchema";
+
+export const getParentRelationship = (peer?: string) => {
+  const peerSchema = useSchema(peer);
```

Comment:
never use hook outside of react. There is a function `getSchema` which is not a hook for this case.